metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1708,7 +1729,10 @@ class MetalsLanguageServer(
     scribe.info(s"memory: $footprint")
   }
 
-  private def indexWorkspace(i: ImportedBuild): Unit = {
+  private var lastImportedBuild = ImportedBuild.empty
```

Comment:
Could we reindex only the Ammonite targets when needed?

Reply:
I don't know… It seems that would require being able to remove particular build targets from the indices (those that changed), before adding them back with updated parameters. I'm not sure if that's do-able.